quotes/tests.py

```diff
@@ -170,3 +197,191 @@ def test_vote_up_pending_fails(self):
         self.client.login(username='test', password='password')
         response = self.client.post(reverse(vote_up, kwargs={'quote_id': 1}))
         self.assertEqual(response.status_code, 404)
+
+    def test_vote_up_GET_405s(self):
+        self.client.login(username='test', password='password')
+        response = self.client.get(reverse(vote_up, kwargs={'quote_id': 2}))
+        self.assertEqual(response.status_code, 405)
```

Comment:
I suggest we use response status codes from the 'status' library, as we don't need to memorize status codes and meanings 😛

Reply:
Agreed.